cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1271,6 +1271,57 @@ def _check_column_availability(self):
                 if "add_labels" in index_def:
                     self._check_single_column_availability(component, index_def["add_labels"])
 
+    def _validate_dense_raw_matrix_values(self) -> None:
+        raw_matrix = self._get_raw_x()
+        for row_index in range(0, self.adata.n_obs):
+            row = raw_matrix[row_index]
+            whole_row_is_zeros = True
+            for col_index, val in enumerate(row):
+                if val:
+                    whole_row_is_zeros = False
+                if not isinstance(val, np.float32) or val != val.astype(int):
+                    self.errors.append(
+                        f"All non-zero values must be positive integers converted to the numpy.float32 type. "
```

Comment:
The error message in `_validate_dense_raw_matrix_values` and `_validate_sparse_raw_matrix_values` are very similar. Can we avoid repeating ourselves. I suggest using something like `"error message {row_index}".format(row_index)`

Reply:
we got feedback from Jason in the [issue](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-curation/614) to fail fast on the first failure rather than find all the rows/indices that are failing, so I'll refactor these error messages accordingly. I'll also incorporate it in the block that does the other raw matrix validations